test/version.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+set -e
+
+: ${IMAGE:?Need to set metrics-server IMAGE variable to test}
```

Comment:
nit: this can be understood as "need to set IMAGE=test", can you reword slightly? same below